tests/test_controller_errors.py

```diff
@@ -108,3 +114,41 @@ def test_bad_orc_checkpoint():
     cont = Controller(launcher="local")
     with pytest.raises(FileNotFoundError):
         cont.reload_saved_db(checkpoint)
+
+
+@pytest.mark.parametrize(
+    "collection,duplicate_name,completed",
```

Comment:
Very pedantic: can we change the name of `collection` to something a bit more representative? Maybe `entity` or `launchable`?